doc/requirements.txt

```diff
@@ -2,3 +2,4 @@ ipython>=7.6.1
 sphinx>=2.1.2
 sphinx_rtd_theme>=0.4.3
 sphinxcontrib-programoutput>=0.14
+click<8.0.0
```

Comment:
Why do we need this restriction? Shouldn't be just click?

Reply:
I also don't get it, probably imcompatible with other package. 
I was trying to build a temporal one (I'm introuducing limix to other people, so need the doc), and I got this error from readthedocs:
```
Installed /home/docs/checkouts/readthedocs.org/user_builds/limix-tempdoc/envs/latest/lib/python3.7/site-packages/threadpoolctl-2.1.0-py3.7.egg
error: click 8.0.1 is installed but click<8.0,>=7.0 is required by {'qtoml'}
```

Reply:
This temporal doc works: https://limix-tempdoc.readthedocs.io
I will use it for now and turn off once you fix the official one. Hope it doesn't introduce too much confusion.